### PR TITLE
change style menu detail page like the coffee house

### PR DESCRIPTION
**Pull request by tri-et**

### CODE REVIEW

File: server/schema/modifier/res.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
 import {Menu} from '../../models'
+import dash from 'lodash'
 const resolvers = {
   RootQuery: {
     async fetchModifiers(_, {input}) {
       var menu = new Menu(input)
```
Comment (hemvanh) [inline server/schema/modifier/res.js]: var --> let

File: server/schema/modifier/res.js [hunk 2]
```diff
@@ -1,9 +1,19 @@
 import {Menu} from '../../models'
+import dash from 'lodash'
```
Comment (hemvanh) [inline server/schema/modifier/res.js]: pls use _ or _d for lodash

File: src/components/ModifiersGroup.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      default: () => [],
+    },
+  },
+  data() {
+    return {
+      modifiers: [],
+    }
+  },
+  filters: {
+    price(val) {
```
Comment (hemvanh) [inline src/components/ModifiersGroup.vue]: price -> formatPrice

File: src/components/ModifiersGroup.vue [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      var price = ''
```
Comment (hemvanh) [inline src/components/ModifiersGroup.vue]: let formatedPrice = ....

File: src/components/ModifiersGroup.vue [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      if (val.includes('-')) {
+        price = '-$' + val.replace('-', '')
+      } else if (val === '0') {
```
Comment (hemvanh) [inline src/components/ModifiersGroup.vue:36]: why price is a string ? val === '0'

File: src/components/ModifiersGroup.vue [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    groupTitle: String,
+    groupType: String,
+    data: {
+      type: Array,
+      default: () => [],
+    },
+  },
+  data() {
+    return {
+      modifiers: [],
```
Comment (hemvanh) [inline src/components/ModifiersGroup.vue]: modifiers -> defaultModifiers

File: src/components/ModifiersGroup.vue [hunk 7]
```diff
@@ -0,0 +1,71 @@
+<template>
+  <div>
+    <div class="title-modify">{{groupTitle}}</div>
+    <q-list inset-separator class="q-pa-none listing-modifier">
+      <q-item v-for="(modifier,index) in data" :key="index">
+        <q-item-main class="text-secondary">
+          <q-radio class="fit" v-if="groupType==='radio'" v-model="modifiers" :val="modifier" :label="modifier.name" color="primary"/>
```
Comment (hemvanh) [inline src/components/ModifiersGroup.vue]: no need to use watch to get defaultModifiers , just set defaultModifiers inside mounted event

File: src/i18n/index.js [hunk 8]
```diff
@@ -1,5 +1,7 @@
 import enUS from './en-us'
+import zhHans from './zh-hans'
```
Comment (hemvanh) [inline src/i18n/index.js]: pls remove testing code

File: src/i18n/zh-hans/index.js [hunk 9]
```diff
@@ -0,0 +1,4 @@
+export default {
```
Comment (hemvanh) [inline src/i18n/zh-hans/index.js]: remove testing code

**Review by hemvanh — CHANGES_REQUESTED** [review]
http://prntscr.com/m1jovr

File: src/components/ModifiersGroup.vue (hunk 5 above)
Comment (tri-et) [reply src/components/ModifiersGroup.vue:36]: type price on db is a string

File: quasar.conf.js [hunk 10]
```diff
@@ -103,12 +103,13 @@ module.exports = function(ctx) {
         'QToggle',
         'QTree',
         'QDialog',
+        'QOptionGroup',
       ],
       directives: ['Ripple', 'CloseOverlay'],
       // Quasar plugins
       plugins: ['Notify', 'AddressbarColor'],
       iconSet: 'material-icons',
```
Comment (hemvanh) [inline quasar.conf.js:111]: i18n : 'de' ?

**Review by hemvanh — CHANGES_REQUESTED** [review]
why seeders need to rename ?

**Pull request merged**